Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -863,6 +863,25 @@ void Window::set_name(String const& name)
     navigable()->active_session_history_entry()->document_state->set_navigable_target_name(name);
 }
 
+// https://html.spec.whatwg.org/multipage/nav-history-apis.html#dom-window-status
+String Window::status() const
+{
+    // When the Window object is created, the attribute must be set to the empty string. It does not do anything else.
+    if (m_status == nullptr) {
```

Comment:
A `String` cannot be `null`, by default it is constructed as the empty `String`.

Reply:
Thank you, looks like that check wasn't needed after all - I have amended my changes :)